utils/clients/tests/test_queue_client.py

```diff
@@ -102,16 +103,29 @@ def test_serialise_message(self):
 
     # pylint:disable=no-self-use
     @patch('stomp.connect.StompConnection11.send')
```

Comment:
Can we change this to mock `utils.clients.queue_client.QueueClient.send`

Reply:
I'm a bit confused by this. `QueueClient.send` is what this test is testing, right? So by patching it out, wouldn't this test nothing except that "send is called when send is called"?

Reply:
Yep you're right ignore me!